src/data/en/index.ts

```diff
@@ -359,13 +357,12 @@ export const instanceData = {
         1060001: 'You successfully recovered your account. Please change your password in the next minutes.',
         1060002: 'An email containing a recovery link has been sent to the email address you provided.',
         1080001: 'An email containing a verification link has been sent to the email address you provided.',
-        1080002: emailVerifiedSuccessfully,
+        1080002: 'You have successfully verified your email address.',
```

Comment:
what do you say to my suggestion to not duplicate this but instead request this string with `…messages.[1080002]` same way you did [here](https://github.com/serlo/frontend/pull/1878/commits/f3a432016a83ca640c6276a09400ec8e675a8e39#diff-a5a83ae18eb89871dba21e44baa7a2df8e0374fed3429ae52ac536cc56bb03afR62)?

Reply:
I actually thought that it was to little to deserve a much code, so I didn't want to spend much time here.
But now I found a solution ([example here](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAlpKBDMwCmARJyYF4YDeAUDDNAE4IDmEAXISaTGCFHOnQ00wPQ-Pg0MEADMYUABZoIQgO4hyAa0bc+MNAFskcADYA1NJRFw0AEwDKAV2AcIIyzp0BPepLgQAdBWqeklyR4a0hBIVNIA2gCMAAwAHNEJAEwAuiq8-Jra+oZwxmZWNsH2ji7wiCjoWMheUJRgNB5+AUEQIWEQUXEJ0SlEaaRqECAwskISSABuQlDDpsNwsJLSckhO-TBhsJm6BkYmFta2xc4AFACUBDDkaFCW5GBl0BWY2Eg1dQ1NEoHBoREx8SSyRgAF91moAEZoYB+GTwMSjGBzcQSdwjVbwWDjCDiYZQmAwxxmGBIHH2VBscDgnjbbJ7fKHIoOU4XK43O4PBBPVAvareeq+fzfFptf5dIGgtIggA0aS+9GI3FIIr+nEVStIAO6iXoAHIAJogSwwcZTMiM1rHJwwKb00wwJxG8jqLS6EmmUzXVoeXWyjVg7gByVgoigSAgHRoDw6EBUE66qGm6gknEAB3IIFThigTl9j2QPKqb35DRYbA4Hlpu1y+wKR2ZTjORD4YaGkejsfjhMj9tJMHJwEpYDzXILlVe7x8HjL7GkldddJrDMKlob5zOQA)) and I'm going to implement it

Reply:
This is what I meant: 
https://github.com/serlo/frontend/pull/1880/commits/b832ae9f5716c415b3beca36ece30c15b99b132a